django_kafka/consumer.py

```diff
@@ -6,6 +6,7 @@
 from confluent_kafka import Consumer as ConfluentConsumer
 from confluent_kafka import cimpl
 
+from django_kafka import producer
```

Comment:
One thing I'm not happy with is that `consumer.py` now imports `producer.py`, seems like a future circular import issue waiting to happen.

I _want_ to put `suppress` within `producers.py` so it's really clear from a decorator perspective:

```
@producer.suppress
def my_func():
    ....
```

But couldn't see another way other than a local import within `.consume()` method.

Reply:
maybe producers can be a module split into files?